### PR TITLE
add performance tab prototype

### PR DESCRIPTION
**Pull request by sturmer-eha**

### CODE REVIEW

File: client/styles/main.styl [hunk 1]
```diff
@@ -61,11 +61,14 @@ nav
     text-indent -9999px
   for $app in $apps
     .{$app}
-      background url('images/'+ $app + '-logo.svg') 40% 50% no-repeat
+      background url('images/' + $app + '-logo.svg') 40% 50% no-repeat
 
   .birt
     background-size 85%
 
+  .performance
```
Comment (Broham) [inline client/styles/main.styl]: @sturmer-eha it looks like @jgoley was defining logos in a slightly different way by adding them to `$apps` on line three and then appending `-logo.svg` up above.  Jonathan - would you like us to continue using this method or is it ok to define logos using this method?

**Comment by Broham** [comment]
This looks good but I did have one question for @jgoley regarding how to configure logos in the navigation tab.  Will hold off until I hear from him to merge.

**Comment by sturmer-eha** [comment]
Let me rebase it meanwhile...

**Comment by jgoley** [comment]
@Broham I suppose it will compile down to the same stuff but, yeah, we should stay consistent.
So @sturmer-eha I think you should just have to change line 3 to:
```$apps = flirt birt performance```
and remove the other stuff

**Comment by jgoley** [comment]
PS: Good catch @Broham 🏈

**Comment by sturmer-eha** [comment]
Should be all done now

File: client/templates/nav.jade [hunk 2]
```diff
@@ -8,3 +8,5 @@ template(name="nav")
         a.birt(href="{{pathFor 'birt'}}") BIRT
       li(class="{{isActiveRoute 'eidr'}}")
         a.eidr(href="{{pathFor 'eidr'}}") EIDR
+      li(class="{{isActiveRoute 'performance'}}")
+        a.performance(href="{{pathFor 'performance'}}") PERFORMANCE
```
Comment (jgoley) [inline client/templates/nav.jade]: I'm not running the application locally but I imagine 'Performance' does not need to be all caps (if so we could use css to transform it)?

File: client/styles/main.styl [hunk 3]
```diff
@@ -72,9 +72,9 @@ nav
     text-indent -9999px
   for $app in $apps
     .{$app}
-      background url('images/'+ $app + '-logo.svg') 40% 50% no-repeat
+      background url('images/' + $app + '-logo.svg') 40% 50% no-repeat
 
-  .birt, .eidr
+  .birt, .eidr, .performance
```
Comment (jgoley) [inline client/styles/main.styl]: The commas aren't necessary. I usually stack them like so:
```
.birt
.eidr
.performance
```

Comment (sturmer-eha) [inline client/styles/main.styl]: I followed the style that was in the file, but, sure, I can change it, no problem.

Comment (jgoley) [inline client/styles/main.styl]: Yeah, I didn't catch it previously. It's not a huge deal.

**Comment by Broham** [comment]
looks good - thanks guys!

**Pull request merged**